solutions/swb-app/src/audit/swbAuditLogger.test.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Writer } from '@aws/workbench-core-audit';
+import SwbAuditLogger from './swbAuditLogger';
+
+describe('SwbAuditLogger', () => {
+  let swbAuditLogger: Writer;
+  beforeEach(() => {
+    swbAuditLogger = new SwbAuditLogger();
+  });
+  describe('write', () => {
+    test('log auditEntry', async () => {
+      // BUILD
+      const consoleSpy = jest.spyOn(global.console, 'log');
+      const auditEntry = {
+        awsRequestId: 'randomAwsRequestId',
+        logEventType: 'audit',
+        action: 'GET /user',
+        actor: { uid: 'sampleID' },
+        source: { ip: 'sampleIP' },
+        body: { name: 'John' }
+      };
+
+      // OPERATE
+      await swbAuditLogger.write({}, auditEntry);
+
+      // CHECK
+      expect(consoleSpy).toBeCalledWith(auditEntry);
```

Comment:
Checking that the method got called means that if the underlying implementation changes and we don't use `global.console` to do logging anymore, this test will break. Is there a way to test inputs and outputs instead?

Reply:
Ah yes, so for this implementation we use `console.log` for logging. If we implement another time of logger, we'll want the unit test to break. We'll want the developer to write a new unit test that captures the new logger being used.